Components/Navigation/index.html

```diff
@@ -0,0 +1,150 @@
+              style="
+                width: 100%;
+                height: 100vh;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+      </main>
+    </div>
+    <script>
+      const hamburgerMenuBtn = document.getElementById("nav-btn");
+      hamburgerMenuBtn.onclick = () => {
+        const navBasicDisplay =
+          document.getElementById("nav-mob").style.display;
+        document.getElementById("nav-mob").style.display =
+          navBasicDisplay === "block" ? "none" : "block";
```

Comment:
You can move this block of code to a js file to avoid using js code in HTML.

Reply:
short code that's why i have written here